subsys/net/lib/nrf_cloud/Kconfig.nrf_cloud_pgps

```diff
@@ -5,7 +5,6 @@
 
 menuconfig NRF_CLOUD_PGPS
 	bool "Enable nRF Cloud Predicted GPS (P-GPS)"
-	depends on NRF_CLOUD_REST || NRF_CLOUD_MQTT
```

Comment:
@tokangas, please check if this removal requires some BUILD_ASSERT etc in Location library

Reply:
I thought that P-GPS needs either MQTT or REST to be enabled, but it looks like it can be used with other transports as well. Currently, Location library supports P-GPS only with MQTT and REST, so either it would be good to a add a BUILD_ASSERT or add support for other transports (like CONFIG_LOCATION_METHOD_GNSS_AGPS_EXTERNAL already does for A-GPS). @hiltunent, what do you think?

Reply:
I don't remember all the details now, but is there genuinely other options for PGPS that fetching the data from nRF Cloud using either MQTT or REST? Isn't this NRF_CLOUD_PGPS_TRANSPORT_NONE essentially there to enable the use of REST?

Reply:
For instance, if the ATv2 is configured for AWS or Azure we use the AWS IoT / Azure MQTT libraries to transport the P-GPS requests/responses.

The P-GPS data is fetched cloud-side from nRF Cloud (using REST) and relayed to the device over the pre-established MQTT connection. In these cases we only use the nRF Cloud P-GPS library to generate the request parameters and process the response.

Reply:
I see. Well then I think best would be to add CONFIG_LOCATION_METHOD_GNSS_PGPS_EXTERNAL. I can't remember why I thought it was not practical when I added CONFIG_LOCATION_METHOD_GNSS_AGPS_EXTERNAL. I'll create a ticket about this.